external/YamlLib.cmake

```diff
@@ -22,13 +22,18 @@
 # Creates ExternalProject for building yaml lib static library
 #
 # Exports:
+#   LIB_STATIC_YAML_INC_LOC: directory of installed yaml lib headers
 #   LIB_STATIC_YAML_LOC: path to installed static yaml lib
 
 
 set(yamllib_url "${REPOSITORY_DIR}/external/common/share/yaml/yaml-0.1.5.tar.gz")
 
 # NOTE Yaml lib doesn't have an install target and leaves artifacts in build dir
 set(yamllib_build_dir "${EP_BASE}/Build/YamlStaticLib")
+set(yamllib_source_dir "${EP_BASE}/Source/YamlStaticLib")
+
+# Export directory to installed static yaml headers to parent
+set(LIB_STATIC_YAML_INC_LOC "${yamllib_source_dir}/include")
```

Comment:
this name doesn't follow our convention for inc dirs by using LOC instead of DIR as suffix. For clarity, `LIB_STATIC_YAML_LOC` points to a specific file, not a directory.